lib_eio/unix/private.ml

```diff
@@ -13,7 +13,6 @@ let await_readable fd = Effect.perform (Await_readable fd)
 let await_writable fd = Effect.perform (Await_writable fd)
 
 let pipe sw = Effect.perform (Pipe sw)
-
```

Comment:
@Arogbonlo Sorry to keep on about this, but you MUST look at your PRs before asking people to review them. Make sure all the changes are supposed to be there.

Reply:
Thank you for this.

But please I'd need your assistance here. Where am I missing it on this change?

Reply:
@talex5  In the previous commit, there was a comment I put there. So the only thing I did here was to delete the comment. 

Except i don't get what you mean. Please I'd like to be clear on it. Thank you!

Reply:
@Arogbonlo -- the change you made here is a deleted newline. Again, this is why reverting your changes (rather than adding a comment and then deleting a comment) would make more sense. We can always squash all of the commits to tidy up the history, but it would be good for the contributor to take care of this.

These kinds of changes (newlines, whitespace etc.) are unnecessary (and add to the review process). As a contributor it is part of your job to also review your own code for these kinds of changes and remove them. Other reviewers want to focus on the _meaningful_ code changes and help you get the logic correct.